README.md

```diff
@@ -53,24 +53,9 @@ The orchestrator can be run using the provided CLI:
 ```bash
 # Run with a pipeline configuration
 node bin/orchestrator.js path/to/your/pipeline.ttl
```

Comment:
This kind of execution belongs under the `## Development` section of the readme.
Here, it should be explained using `npx rdfc ...`

Additionally, idem for the Installation part above: that are installation instructions for if you wish to develop on the orchestrator, and should move to the development section below.
The first thing that readers should read in the readme is how to install the orchestrator in order to use it in their pipelines, thus the installation instructions should be updated with `npm i @rdfc/orchestrator-js`